diff --git a/dist/runtime/plugin.client.mjs b/dist/runtime/plugin.client.mjs
index 1f80861124b897e18e0089dbd39d3a7e2b308df6..940799ad052778fa1ff4c3daafad2a29dd1e6ae9 100644
--- a/dist/runtime/plugin.client.mjs
+++ b/dist/runtime/plugin.client.mjs
@@ -77,9 +77,9 @@ export default defineNuxtPlugin((nuxtApp) => {
     watchMedia();
   }
   nuxtApp.hook("app:mounted", () => {
-    if (window.localStorage) {
-      watchStorageChange();
-    }
+    // if (window.localStorage) {
+    //   watchStorageChange();
+    // }
     if (colorMode.unknown) {
       colorMode.preference = helper.preference;
       colorMode.value = helper.value;